### PR TITLE
New application yt-dlp.yt-dlp version 2022.05.18

### LINKED ISSUE #35475: [Package Request]: yt-dlp
### Package Requested

- [X] I would like someone else to build the manifest.
- [ ] I would like help so I can submit the manifest.
- [X] I have performed a search and couldn't find this package.
- [X] I have checked that the installer for this package is not a standalone or portable executable or a .zip file
- [ ] I think there is a new version available and I have provided the URL.

### Please provide the following information

Publisher: yt-dlp
Publisher Url: https://github.com/yt-dlp/yt-dlp/releases
Package Name: yt-dlp
Description: A youtube-dl fork with additional features and fixes
Package Version: 2021.11.10.1
Installer URL: https://github.com/yt-dlp/yt-dlp/releases/download/2021.11.10.1/yt-dlp.exe

Current Package Identifier (for new versions):

### PR DESCRIPTION
**Pull request by KaranKad**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Resolves https://github.com/microsoft/winget-pkgs/issues/35475

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/61435)

### CODE REVIEW

**Review by ItzLevvie — APPROVED** [review]

**Comment by KaranKad** [comment]
@denelon add blocking issue exe

**Comment by KaranKad** [comment]
Bump

File: manifests/y/yt-dlp/yt-dlp/2022.05.18/yt-dlp.yt-dlp.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: yt-dlp.yt-dlp
```
Comment (lechacon) [inline manifests/y/yt-dlp/yt-dlp/2022.05.18/yt-dlp.yt-dlp.installer.yaml:3]: #68801 is adding another version of this package, and using the identifier YT.DLP. Looks like that's what we're going with.

**Review by florelis — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request closed without merging**